### PR TITLE
Release 2016-08-03 19:40:36

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/Controller/PagesController.php [hunk 1]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
  * @var array
  */
 	public $uses = array();
+	public $uses2 = array();
```
Comment (sweep3092) [inline app/Controller/PagesController.php:39]: **[staging][code_sniffer]**
- Spaces must be used to indent lines; tabs are not allowed
- Line indented incorrectly; expected 4 spaces, found 1

File: app/Controller/PagesController.php [hunk 2]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+	public $uses3 = array();
```
Comment (sweep3092) [inline app/Controller/PagesController.php:40]: **[staging][code_sniffer]**
- Spaces must be used to indent lines; tabs are not allowed
- Line indented incorrectly; expected 4 spaces, found 1

File: app/Controller/PagesController.php [hunk 3]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+
+/**
```
Comment (sweep3092) [inline app/Controller/PagesController.php:42]: **[staging][code_sniffer]**
- Line indented incorrectly; expected at least 4 spaces, found 0

File: app/Controller/PagesController.php [hunk 4]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+ * TooManyMethods
+*/
```
Comment (sweep3092) [inline app/Controller/PagesController.php:44]: **[staging][code_sniffer]**
- Expected 1 space(s) before asterisk; 0 found
- Missing @return tag in function comment

File: app/Controller/PagesController.php [hunk 5]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+public function display1() {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:45]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected 2 blank lines before function; 1 found
- Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 6]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+}
```
Comment (sweep3092) [inline app/Controller/PagesController.php:46]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display1()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php [hunk 7]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+public function display2() {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:47]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 8]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+}
```
Comment (sweep3092) [inline app/Controller/PagesController.php:48]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display2()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php [hunk 9]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+public function display3() {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:49]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 10]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+}
```
Comment (sweep3092) [inline app/Controller/PagesController.php:50]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display3()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php [hunk 11]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+public function display4() {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:51]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 12]
```diff
@@ -45,6 +85,12 @@ class PagesController extends AppController {
  *	or MissingViewException in debug mode.
  */
 	public function display() {
+    $eeeeeeeeee;
```
Comment (ghost) [inline app/Controller/PagesController.php:88]: **[phpmd]**
- Avoid unused local variables such as '$eeeeeeeeee'.

File: app/Controller/PagesController.php [hunk 13]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+ * TooManyMethods
+*/
+public function display1() {
+}
+public function display2() {
+}
+public function display3() {
+}
+public function display4() {
+}
```
Comment (sweep3092) [inline app/Controller/PagesController.php:52]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display4()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php [hunk 14]
```diff
@@ -45,6 +85,12 @@ class PagesController extends AppController {
  *	or MissingViewException in debug mode.
  */
 	public function display() {
+    $eeeeeeeeee;
+    $aaaaaaaaaa;
+    $bbbbbbbbbb;
```
Comment (ghost) [inline app/Controller/PagesController.php:90]: **[phpmd]**
- Avoid unused local variables such as '$bbbbbbbbbb'.

File: app/Controller/PagesController.php (hunk 12 above)
Comment (sweep3092) [inline app/Controller/PagesController.php:88]: **[staging][phpmd]**
- Avoid unused local variables such as '$eeeeeeeeee'.

File: app/Controller/PagesController.php [hunk 15]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+*/
+public function display1() {
+}
+public function display2() {
+}
+public function display3() {
+}
+public function display4() {
+}
+public function display5() {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:53]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 16]
```diff
@@ -45,6 +85,12 @@ class PagesController extends AppController {
  *	or MissingViewException in debug mode.
  */
 	public function display() {
+    $eeeeeeeeee;
+    $aaaaaaaaaa;
+    $bbbbbbbbbb;
+    $aaaaaaaaaa;
+
+    $cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc;
```
Comment (ghost) [inline app/Controller/PagesController.php:93]: **[phpmd]**
- Avoid excessively long variable names like $cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc. Keep variable name length under 20.
- Avoid unused local variables such as '$cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc'.

File: app/Controller/PagesController.php [hunk 17]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+public function display1() {
+}
+public function display2() {
+}
+public function display3() {
+}
+public function display4() {
+}
+public function display5() {
+}
```
Comment (sweep3092) [inline app/Controller/PagesController.php:54]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display5()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php (hunk 14 above)
Comment (sweep3092) [inline app/Controller/PagesController.php:90]: **[staging][phpmd]**
- Avoid unused local variables such as '$bbbbbbbbbb'.

File: app/Controller/PagesController.php [hunk 18]
```diff
@@ -72,5 +118,59 @@ public function display() {
 			}
 			throw new NotFoundException();
 		}
-	}
+  }
+
+  private $顧客名;
+
+  function set顧客名($v)
```
Comment (ghost) [inline app/Controller/PagesController.php:125]: **[phpmd]**
- The method set顧客名 is not named in camelCase.
- Avoid variables with short names like $v. Configured minimum length is 3.

File: app/Controller/PagesController.php [hunk 19]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+}
+public function display2() {
+}
+public function display3() {
+}
+public function display4() {
+}
+public function display5() {
+}
+public function display6() {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:55]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php (hunk 16 above)
Comment (sweep3092) [inline app/Controller/PagesController.php:93]: **[staging][phpmd]**
- Avoid excessively long variable names like $cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc. Keep variable name length under 20.
- Avoid unused local variables such as '$cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc'.

File: app/Controller/PagesController.php [hunk 20]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+}
```
Comment (sweep3092) [inline app/Controller/PagesController.php:56]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display6()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php (hunk 18 above)
Comment (sweep3092) [inline app/Controller/PagesController.php:125]: **[staging][phpmd]**
- The method set顧客名 is not named in camelCase.
- Avoid variables with short names like $v. Configured minimum length is 3.

File: app/Controller/PagesController.php [hunk 21]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+public function display7() {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:57]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 22]
```diff
@@ -72,5 +118,59 @@ public function display() {
...
+
+  private $顧客名;
+
+  function set顧客名($v)
+  {
+    $this->顧客名 = $v;
+    return $this;
+  }
+
+  function get顧客名()
```
Comment (ghost) [inline app/Controller/PagesController.php:131]: **[phpmd]**
- The method get顧客名 is not named in camelCase.

Comment (sweep3092) [inline app/Controller/PagesController.php:131]: **[staging][phpmd]**
- The method get顧客名 is not named in camelCase.

File: app/Controller/PagesController.php [hunk 23]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+public function display3() {
+}
+public function display4() {
+}
+public function display5() {
+}
+public function display6() {
+}
+public function display7() {
+}
```
Comment (sweep3092) [inline app/Controller/PagesController.php:58]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display7()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php [hunk 24]
```diff
@@ -72,5 +118,59 @@ public function display() {
...
+    return $this->顧客名;
+  }
+}
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
```
Comment (ghost) [inline app/Controller/PagesController.php:142]: **[phpmd]**
- The method example() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10.

Comment (sweep3092) [inline app/Controller/PagesController.php:142]: **[staging][phpmd]**
- The method example() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10.

File: app/Controller/PagesController.php [hunk 25]
```diff
@@ -72,5 +118,59 @@ public function display() {
...
+        if ($a == $b)  {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:143]: **[staging][phpmd]**
- Avoid variables with short names like $a. Configured minimum length is 3.
- Avoid variables with short names like $b. Configured minimum length is 3.
- Avoid unused local variables such as '$a'.
- Avoid unused local variables such as '$b'.

File: app/Controller/PagesController.php [hunk 26]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+}
+public function display4() {
+}
+public function display5() {
+}
+public function display6() {
+}
+public function display7() {
+}
+public function display8() {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:59]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php (hunk 25 above)
Comment (ghost) [inline app/Controller/PagesController.php:143]: **[phpmd]**
- Avoid variables with short names like $a. Configured minimum length is 3.
- Avoid variables with short names like $b. Configured minimum length is 3.
- Avoid unused local variables such as '$a'.
- Avoid unused local variables such as '$b'.

File: app/Controller/PagesController.php [hunk 27]
```diff
@@ -72,5 +118,59 @@ public function display() {
...
+}
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
+        if ($a == $b)  {
+            if ($a1 == $b1) {
```
Comment (ghost) [inline app/Controller/PagesController.php:144]: **[phpmd]**
- Avoid variables with short names like $a1. Configured minimum length is 3.
- Avoid variables with short names like $b1. Configured minimum length is 3.
- Avoid unused local variables such as '$a1'.
- Avoid unused local variables such as '$b1'.

File: app/Controller/PagesController.php [hunk 28]
```diff
@@ -72,5 +118,59 @@ public function display() {
...
+                fiddle();
+            } else if ($a2 == $b2) {
```
Comment (ghost) [inline app/Controller/PagesController.php:146]: **[phpmd]**
- Avoid variables with short names like $a2. Configured minimum length is 3.
- Avoid variables with short names like $b2. Configured minimum length is 3.
- Avoid unused local variables such as '$a2'.
- Avoid unused local variables such as '$b2'.

File: app/Controller/PagesController.php [hunk 29]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+public function display4() {
+}
+public function display5() {
+}
+public function display6() {
+}
+public function display7() {
+}
+public function display8() {
+}
```
Comment (sweep3092) [inline app/Controller/PagesController.php:60]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display8()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php (hunk 27 above)
Comment (sweep3092) [inline app/Controller/PagesController.php:144]: **[staging][phpmd]**
- Avoid variables with short names like $a1. Configured minimum length is 3.
- Avoid variables with short names like $b1. Configured minimum length is 3.
- Avoid unused local variables such as '$a1'.
- Avoid unused local variables such as '$b1'.

File: app/Controller/PagesController.php [hunk 30]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+public function display9() {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:61]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 31]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+}
```
Comment (sweep3092) [inline app/Controller/PagesController.php:62]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display9()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php (hunk 28 above)
Comment (sweep3092) [inline app/Controller/PagesController.php:146]: **[staging][phpmd]**
- Avoid variables with short names like $a2. Configured minimum length is 3.
- Avoid variables with short names like $b2. Configured minimum length is 3.
- Avoid unused local variables such as '$a2'.
- Avoid unused local variables such as '$b2'.

File: app/Controller/PagesController.php [hunk 32]
```diff
@@ -72,5 +118,59 @@ public function display() {
...
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
+        if ($a == $b)  {
+            if ($a1 == $b1) {
+                fiddle();
+            } else if ($a2 == $b2) {
+                fiddle();
+            }  else {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:148]: **[staging][phpmd]**
- The method example uses an else expression. Else is never necessary and you can simplify the code to work without else.

File: app/Controller/PagesController.php [hunk 33]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+}
+public function display6() {
+}
+public function display7() {
+}
+public function display8() {
+}
+public function display9() {
+}
+public function display10() {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:63]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php [hunk 34]
```diff
@@ -72,5 +118,59 @@ public function display() {
...
+    public function example()  {
+        if ($a == $b)  {
+            if ($a1 == $b1) {
+                fiddle();
+            } else if ($a2 == $b2) {
+                fiddle();
+            }  else {
+                fiddle();
+            }
+        } else if ($c == $d) {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:151]: **[staging][phpmd]**
- Avoid variables with short names like $c. Configured minimum length is 3.
- Avoid variables with short names like $d. Configured minimum length is 3.

File: app/Controller/PagesController.php [hunk 35]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+public function display6() {
+}
+public function display7() {
+}
+public function display8() {
+}
+public function display9() {
+}
+public function display10() {
+}
```
Comment (sweep3092) [inline app/Controller/PagesController.php:64]: **[staging][code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display10()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php [hunk 36]
```diff
@@ -72,5 +118,59 @@ public function display() {
...
+            } else if ($a2 == $b2) {
+                fiddle();
+            }  else {
+                fiddle();
+            }
+        } else if ($c == $d) {
+            while ($c == $d) {
+                fiddle();
+            }
+         } else if ($e == $f) {
```
Comment (sweep3092) [inline app/Controller/PagesController.php:155]: **[staging][phpmd]**
- Avoid variables with short names like $e. Configured minimum length is 3.
- Avoid variables with short names like $f. Configured minimum length is 3.
- Avoid unused local variables such as '$e'.
- Avoid unused local variables such as '$f'.

File: app/Controller/PagesController.php (hunk 1 above)
Comment (ghost) [inline app/Controller/PagesController.php:39]: **[code_sniffer]**
- Spaces must be used to indent lines; tabs are not allowed
- Line indented incorrectly; expected 4 spaces, found 1

File: app/Controller/PagesController.php (hunk 2 above)
Comment (ghost) [inline app/Controller/PagesController.php:40]: **[code_sniffer]**
- Spaces must be used to indent lines; tabs are not allowed
- Line indented incorrectly; expected 4 spaces, found 1

File: app/Controller/PagesController.php (hunk 3 above)
Comment (ghost) [inline app/Controller/PagesController.php:42]: **[code_sniffer]**
- Line indented incorrectly; expected at least 4 spaces, found 0

File: app/Controller/PagesController.php (hunk 4 above)
Comment (ghost) [inline app/Controller/PagesController.php:44]: **[code_sniffer]**
- Expected 1 space(s) before asterisk; 0 found
- Missing @return tag in function comment

File: app/Controller/PagesController.php (hunk 5 above)
Comment (ghost) [inline app/Controller/PagesController.php:45]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected 2 blank lines before function; 1 found
- Opening brace should be on a new line

File: app/Controller/PagesController.php (hunk 6 above)
Comment (ghost) [inline app/Controller/PagesController.php:46]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display1()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php (hunk 7 above)
Comment (ghost) [inline app/Controller/PagesController.php:47]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php (hunk 8 above)
Comment (ghost) [inline app/Controller/PagesController.php:48]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display2()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php (hunk 9 above)
Comment (ghost) [inline app/Controller/PagesController.php:49]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php (hunk 10 above)
Comment (ghost) [inline app/Controller/PagesController.php:50]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display3()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php (hunk 11 above)
Comment (ghost) [inline app/Controller/PagesController.php:51]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php (hunk 13 above)
Comment (ghost) [inline app/Controller/PagesController.php:52]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display4()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php (hunk 15 above)
Comment (ghost) [inline app/Controller/PagesController.php:53]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php (hunk 17 above)
Comment (ghost) [inline app/Controller/PagesController.php:54]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display5()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php (hunk 19 above)
Comment (ghost) [inline app/Controller/PagesController.php:55]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php (hunk 20 above)
Comment (ghost) [inline app/Controller/PagesController.php:56]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display6()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php (hunk 21 above)
Comment (ghost) [inline app/Controller/PagesController.php:57]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php (hunk 23 above)
Comment (ghost) [inline app/Controller/PagesController.php:58]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display7()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php (hunk 26 above)
Comment (ghost) [inline app/Controller/PagesController.php:59]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php (hunk 29 above)
Comment (ghost) [inline app/Controller/PagesController.php:60]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display8()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php (hunk 30 above)
Comment (ghost) [inline app/Controller/PagesController.php:61]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php (hunk 31 above)
Comment (ghost) [inline app/Controller/PagesController.php:62]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display9()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php (hunk 33 above)
Comment (ghost) [inline app/Controller/PagesController.php:63]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

File: app/Controller/PagesController.php (hunk 35 above)
Comment (ghost) [inline app/Controller/PagesController.php:64]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Expected //end display10()
- Expected 1 blank line before closing function brace; 0 found
- Expected 2 blank lines after function; 0 found

File: app/Controller/PagesController.php [hunk 37]
```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
...
+}
+public function display7() {
+}
+public function display8() {
+}
+public function display9() {
+}
+public function display10() {
+}
+public function display11() {
```
Comment (ghost) [inline app/Controller/PagesController.php:65]: **[code_sniffer]**
- Line indented incorrectly; expected 4 spaces, found 0
- Missing function doc comment
- Opening brace should be on a new line

**Pull request closed without merging**